public/scripts.js

```diff
@@ -33,6 +33,11 @@ document.addEventListener('DOMContentLoaded', function() {
       author.innerText = "– " + randomQuote.author;
       blockquote.classList.remove("flipOutX");
       blockquote.classList.add("flipInX");
+      
+      var quoteClass = quote.getAttribute("class")
+      if (quoteClass.indexOf("notation") == -1) {
```

Comment:
There's this stupid JavaScript equality thing, btw. The preferred way to check is with a triple equals (`===`), since that also matches on type. Basically:

```javascript
-1 == '-1'  // => true
-1 === '-1' // => false
```
It doesn't really matter in this case, because it's not gonna be a string, but it's preferred. :wink: